bin/update

```diff
@@ -16,7 +16,10 @@ end
 
 Dir.chdir APP_ROOT do
   puts "== Installing dependencies =="
-  execute "bundle update"
+  execute('gem install bundler --conservative')
+
+  # Swallow non-zero bundle check exit codes and run install
+  system('bundle check') || execute('bundle install')
```

Comment:
Okay, I may have lied at little...

So, I like this from a speed perspective, but I feel like we will get out of sync between devs and travis.  An example scenario:

* Travis runs a build installs Rails at, lets say, `5.0.0.1`
* Developer runs `bin/update`, installs `5.0.0.1`
* Rails updates a minor version
* Developer runs `bin/update`, nothing changes, runs tests, all pass
* Travis runs, installs the new version of Rails, test fail do to change in functionality

Or at least, I think this is what would happen.  I will say, this is probably more of the quick part of the script, so this maybe isn't something worth changing.